### DISCUSSION TITLE
Is it possible to insert LaTeX inside the caption of a figure?

### DISCUSSION THREAD

**Discussion by choisy** [quarto-dev/quarto-cli#8628] [Q&A]
### Description

I would like to do something like this

```{r}
#| label: fig-example
#| fig-cap: "The caption with some LaTeX like this $\lambda$."

plot(1:10)
```

**Comment by mcanouil** [comment]
Did you try to do what you are asking?

Please provide a complete reproducible example that include format(s), etc.

See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Comment by cderv** [comment]
@choisy you probably need to leverage the new Div syntax for this. The caption is directly processed as Markdown content in that case when provided inside the div below the code chunk. 

https://quarto.org/docs/authoring/cross-references-divs.html

````markdown
---
title: pdf caption
format: pdf
---

::: {#fig-example}

```{r}
#| echo: false
plot(1:10)
```

The caption with some LaTeX like this $\lambda$.
:::

````

![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/1c53bf1f-5534-47c0-87c8-f7c3630c98aa)

**Reply by mcanouil** [reply]
`*-cap` will work just fine:

````qmd
---
format: pdf
---

```{r}
#| label: fig-example
#| fig-cap: "The caption with some LaTeX like this $\\lambda$."
plot(1:10)
```
````

- https://github.com/quarto-dev/quarto-cli/discussions/7591
- https://github.com/quarto-dev/quarto-cli/discussions/4756
- https://github.com/quarto-dev/quarto-cli/discussions/6024
- https://github.com/quarto-dev/quarto-cli/discussions/1785
